scripts/publish-apt-packages.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# This script is used to publish new packages to the CLI APT repository
+# Usage: ./publish-apt-packages.sh
+set -eo pipefail
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+OBJECT_STORAGE_ENDPOINT="https://object.storage.eu01.onstackit.cloud"
+APT_BUCKET_NAME="stackit-cli-apt"
+PUBLIC_KEY_BUCKET_NAME="stackit-public-key"
+PUBLIC_KEY_FILE="key.gpg"
+CUSTOM_KEYRING="custom-keyring"
+APTLY_CONFIG_FILE_PATH="./.aptly.conf"
+
+# Create a local mirror of the current state of the remote APT repository
+printf ">>> Creating mirror \n"
+curl ${OBJECT_STORAGE_ENDPOINT}/${PUBLIC_KEY_BUCKET_NAME}/${PUBLIC_KEY_FILE} >public.asc
+gpg --no-default-keyring --keyring ./${CUSTOM_KEYRING}.gpg --import public.asc
+aptly mirror create -keyring="${CUSTOM_KEYRING}.gpg" current "${OBJECT_STORAGE_ENDPOINT}/${APT_BUCKET_NAME}" stackit
+
+# Update the mirror to the latest state
+printf "\n>>> Updating mirror \n"
+aptly mirror update current
+
+# Create a snapshot of the mirror
+printf "\n>>> Creating snapshop from mirror \n"
+aptly snapshot create current-snapshot from mirror current
+
+# Create a new fresh local APT repo
+printf "\n>>> Creating fresh local repo \n"
+aptly repo create -distribution="stackit-cli" new-repo
+
+# Add new generated .deb packages to the new local repo
+printf "\n>>> Adding new packages to local repo \n"
+aptly repo add new-repo dist/
```

Comment:
where is this `dist/` folder coming from?

Reply:
When GoReleaser is run it creates that folder if it doesn't exist, and that is where it will store all the generated packages/binaries

Reply:
I will add a comment to clarify